### PR TITLE
fix: updated permissions flow copy changes

### PR DESCRIPTION
**Pull request by NidhiKJha**
This PR is to update the copy changes in Permissions Screen

## **Related issues**

Fixes: [https://github.com/MetaMask/MetaMask-planning/issues/3391](https://github.com/MetaMask/MetaMask-planning/issues/3391)

## **Manual testing steps**

1. Run extension with yarn start
2. For all permissions page, there should be no extra space between "[x] accounts" and the "•" in the list item description
3. Click on connected permission, On Permissions Page check 4px spacing between the Permission label and the secondary description
4. Click Disconnect button and the Modal copy should be "If you disconnect from this site, you’ll need to reconnect your accounts and networks to use this site again."
5. When no account is connected, copy should be updated as well

For tooltips, I will file a separate issue and we are not adding any badge for avatars on permissions page

### **After**
Screenshot 2024-10-07 at 1 35 23 PM

Screenshot 2024-10-07 at 1 36 35 PM

Screenshot 2024-10-07 at 1 36 47 PM
Screenshot 2024-10-07 at 1 37 12 PM

## **Pre-merge author checklist**
- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by andreahaku — DISMISSED** [review]
LGTM

**Review by andreahaku — DISMISSED** [review]
LGTM

**Review by andreahaku — APPROVED** [review]
LGTM

File: ui/components/multichain/pages/review-permissions-page/review-permissions-page.tsx [hunk 1]
```diff
@@ -265,14 +265,18 @@ export const ReviewPermissions = () => {
...
             ) : (
-              <ButtonPrimary
-                size={ButtonPrimarySize.Lg}
-                block
-                data-test-id="no-connections-button"
-                onClick={requestAccountsAndChainPermissions}
-              >
-                {t('connectAccounts')}
-              </ButtonPrimary>
+              <>
```
Comment (jonybur) [inline ui/components/multichain/pages/review-permissions-page/review-permissions-page.tsx:268]: Not a blocker but I think this Fragment is not necessary

**Review by jonybur — APPROVED** [review]

**Pull request merged**